### DISCUSSION TITLE
About API usage

### DISCUSSION THREAD

**Discussion by NYT92** [amvstrm/amvstrm#44] [Announcements]
Hello, everyone. So there are many people who use my API right now and also there are some people who use our production API without my knowledge leading to over 200K requests (cloudflare).

Just remembered, both pub API and prod API are hosted on a serverless hosting provider like vercel with low amount of resources putting behind cloudflare. I do not own a server and I don't logged those requests because it too expensive.

I provided an API and open-sources it for free and you can easily hosted it anywhere you want. [More info](https://docs.amvstr.me/docs/api/introduction)

I might temporarily closed the API from public access any moment not because I'm sick of the ddos because I am struggling with school right now.

From now on, if anyone trying to access production API will be immediately blocked by us. Just host your own you dip shit

**Comment by JammyVibez** [comment]
Hello I need your API and streaming service
